packages/design-system-react-native/src/components/Button/variants/ButtonSecondary/ButtonSecondary.tsx

```diff
@@ -18,62 +17,65 @@ export const ButtonSecondary = ({
   isDanger = false,
   isInverse = false,
   isLoading = false,
-  onPressIn,
-  onPressOut,
   twClassName = '',
   style,
   ...props
 }: ButtonSecondaryProps) => {
-  const [isPressed, setIsPressed] = useState(false);
   const tw = useTailwind();
-  const twContainerClassNames = `
-    ${
-      isInverse && isDanger
-        ? isPressed || isLoading
-          ? 'bg-default-pressed'
-          : 'bg-default'
-        : isDanger
-          ? isPressed || isLoading
-            ? 'bg-error-muted-pressed'
-            : 'bg-transparent'
-          : isPressed || isLoading
+
+  const getContainerClassName = useCallback(
+    (pressed: boolean): string => {
+      const baseClasses = `
+        border-[1.5px]
```

Comment:
Is this border value correct? I see in the Figma that it is one pixel. cc @Akatori-Design 

<img width="1104" alt="Screenshot 2025-06-11 at 12 01 17 PM" src="https://github.com/user-attachments/assets/1c156b51-f9bd-4703-b014-3387e26d0f34" />